test/resources/deliusoffender/offender.json

```diff
@@ -113,7 +113,6 @@
     "immigrationStatus": "string",
     "nationality": "British",
     "notes": "string",
-    "offenderDetails": {},
```

Comment:
All these json files were wrong; the `offenderDetails` note (aka notes) is a string not an object.